### PR TITLE
Add removeItem term entry

### PR DESCRIPTION
**Pull request by braanj**
### Type of Change

Add the removeItem term entry to javascript.

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by braanj** [comment]
@SSwiniarski Hello, plz review my PR

File: content/javascript/concepts/storage/terms/removeItem/removeItem.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+Title: 'removeItem()'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/removeItem/removeItem.md:2]:
```suggestion
Title: '.removeItem()'
```

File: content/javascript/concepts/storage/terms/removeItem/removeItem.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**`.removeItem()`** is a method provided by the Web Storage API in JavaScript. It allows you to remove a specific key-value pair from the local storage or session storage based on the specified key.
+
+## Syntax
+
+When called, the `removeItem()` method takes a single parameter, which is the key of the item to be removed. It searches for the key in the storage and, if found, removes the associated key-value pair from the storage. If the key does not exist, the method does nothing and the storage remains unchanged.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/removeItem/removeItem.md:20]:
```suggestion
When called, the `.removeItem()` method takes a single parameter, `keyName`, which is the key of the item to be removed. The function removes the associated key-value pair from storage, if found. If the key does not exist, the method does nothing and the storage remains unchanged.
```

Comment (caupolicandiaz) [reply content/javascript/concepts/storage/terms/removeItem/removeItem.md:20]: I'd recommend putting the description after the syntax block

File: content/javascript/concepts/storage/terms/removeItem/removeItem.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+```pseudo
+localStorage.removeItem('keyName');
+```
+
+## Example
+
+The following code demonstrates the application of `removeItem()` on the localeStorage:
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/removeItem/removeItem.md:28]:
```suggestion
The following code demonstrates the application of `.removeItem()` to modify `localStorage`:
```

File: content/javascript/concepts/storage/terms/removeItem/removeItem.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+```js
+localStorage.setItem('username', 'JohnDoe');
+
+localStorage.removeItem('username');
+```
+
+In the example above, a key-value pair (`username`, `JohnDoe`) is added to the localeStorage using `setItem()`, and removed using `removeItem()`.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/removeItem/removeItem.md:36]:
```suggestion
In the example above, a key-value pair (`username`, `JohnDoe`) is added to the `localStorage` using `.setItem()`, and removed using `.removeItem()`.
```

File: content/javascript/concepts/storage/terms/removeItem/removeItem.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+Similarly, you can do the same for the session storage:
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/removeItem/removeItem.md:38]:
```suggestion
The same can be done with the session storage:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@braanj I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by braanj** [comment]
@caupolicandiaz Thanks for the review. Changes made!

**Review by caupolicandiaz — APPROVED** [review]
@braanj This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@braanj this looks goo to merge once the "Storage" entry is added.

**Comment by braanj** [comment]
@SSwiniarski Thanks for the review

**Comment by SSwiniarski** [comment]
@braanj Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/javascript/storage/removeItem

image

**Pull request merged**